docs/rules/arrow-parens.md

```diff
@@ -12,39 +12,45 @@ This rule ignores arrow functions that there is a open paren before itself.
 ### The following patterns are considered warnings:
 
 ```js
-const twice = x => 2 * x;
+const twice = (x) => 2 * x
```

Comment:
I think this file should probably not be formatted.

Reply:
This is because I ran `prettier` on all files, so I think it's OK

Reply:
Formatting the example code in this file would not be a good explanation of the rule.
This should explain the violation of this rule related to parentheses.

Reply:
Oh your talking about this specific file, I thought you were talking about all doc files 🙈

Will revert 👍